standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -243,6 +238,18 @@ Each term in this vocabulary has a human-readable description and may include ad
 ***** Shoulder <nowiki>{hedId=HED_0012227} [Joint attaching upper arm to trunk.]</nowiki>
 ***** Upper-arm <nowiki>{hedId=HED_0012228} [Portion of arm between shoulder and elbow.]</nowiki>
 ***** Wrist <nowiki>{hedId=HED_0012229} [A joint between the distal end of the radius and the proximal row of carpal bones.]</nowiki>
+**** Lower-extremity <nowiki>{hedId=HED_0012195} [Refers to the whole inferior limb (leg and/or foot).]</nowiki>
+***** Ankle <nowiki>{hedId=HED_0012196} [A gliding joint between the distal ends of the tibia and fibula and the proximal end of the talus.]</nowiki>
+***** Calf <nowiki>{hedId=HED_0012197} [The fleshy part at the back of the leg below the knee.]</nowiki>
+***** Foot <nowiki>{hedId=HED_0012198} [The structure found below the ankle joint required for locomotion.]</nowiki>

```

Comment:
Do we need Foot-part.

Reply:
I don't quite understand the question. Foot part is on the line below.